src/appservice/AppService.ts

```diff
@@ -75,21 +78,35 @@ export class MjolnirAppService {
             // It also allows us to combine constructor/initialize logic
             // to make the code base much simpler. A small hack to pay for an overall less hacky code base.
             controller: {
-                onUserQuery: () => {throw new Error("Mjolnir uninitialized")},
-                onEvent: () => {throw new Error("Mjolnir uninitialized")},
+                onUserQuery: () => { throw new Error("Mjolnir uninitialized") },
+                onEvent: () => { throw new Error("Mjolnir uninitialized") },
             },
             suppressEcho: false,
         });
         await bridge.initialise();
         const accessControlListId = await bridge.getBot().getClient().resolveRoom(config.adminRoom);
         const accessControl = await AccessControl.setupAccessControl(accessControlListId, bridge);
-        const mjolnirManager = await MjolnirManager.makeMjolnirManager(dataStore, bridge, accessControl);
+        // Activate /metrics endpoint for Prometheus
+
+        // This should happen automatically but in testing this didn't happen in the docker image
+        setBridgeVersion(SOFTWARE_VERSION);
```

Comment:
Are you happy that this can say "A version was either not provided when building Draupnir or could not be read." in an error state? If so that's cool, just checking you knew that's what it does.

Reply:
I think its fine. In the end its just a string thats mandatory by the bridge when using the metrics stuff. It actually is only called because for some reason the path it checks in the bridge sdk is not pointing to the package.json when draupnir is in docker. (It does work when outside of docker though)